site/_includes/partials/toc-under-tree.njk

```diff
@@ -0,0 +1,11 @@
+{#
+  This is used as the TOC on smaller screens (<= 992) that makes use of the white space
+  under the navigation tree when the user scrolls down, in response to:
+  https://github.com/GoogleChrome/developer.chrome.com/issues/3097
+#}
+<div class="toc-container toc-container-under-navigation-tree display-none md:display-block xl:display-none pad-right-400 {% if not tocContents %}visibility-hidden{% endif %}">
```

Comment:
This is repeating the content of toc-side.njk

Can we reuse the same template, and modify display properties in the css class? 

Also, we generally use BEM notation for non-utility classes. So this class should be something like toc-container--sticky or so. However, since this class is used mostly for positioning of the element, it would be better to add a css rule in side-nav instead (https://github.com/GoogleChrome/developer.chrome.com/blob/main/site/_scss/blocks/_side-nav.scss), e.g. 

navigation-tree toc-container {...}